docs/current_docs/api/module-dependencies.mdx

```diff
@@ -107,3 +109,39 @@ dagger install ./path/to/module
 :::note
 Installing a module using a local path (relative or absolute) is only possible if your module is within the repository root (for Git repositories) or the directory containing the `dagger.json` file (for all other cases).
 :::
+
+## Uninstallation
+
+To remove a dependency from your Dagger module, use the `dagger uninstall` command, as in the following example:
+
+```sh
+dagger uninstall github.com/shykes/daggerverse/hello
+```
+
+The `dagger uninstall` command can be passed either a remote repository reference or a local module name. If a version number is specified, only the specified version of the dependency is uninstalled.
```

Comment:
I don't quite follow the use case here. A dependency only has one version. When would you want to uninstall a specific version? Even if someone needs to uninstall that specific version in a script or something, but otherwise leave it be, seems to me this would require manual intervention. I may not be seeing the right use case, but it seems edge casey enough that I'd suggest simplifying this here by omitting this feature. Just uninstall by name.

Reply:
Updated